draft-irtf-cfrg-vdaf.md

```diff
@@ -3321,12 +3321,32 @@ struct {
 } Prio3AggShare;
 ~~~
 
-## The FLP Construction {#flp-bbcggi19}
+## FLP Construction {#flp-bbcggi19}
 
-This section describes an FLP based on the construction from {{BBCGGI19}},
-Section 4.2. We begin in {{flp-bbcggi19-overview}} with an overview of their
-proof system and some extensions to it. The construction is specified in
-{{flp-bbcggi19-construction}}.
+This section specifies an implementation of the `Flp` interface ({{flp}}) based
+on the construction from {{BBCGGI19}}, Section 4.2. We begin in
+{{flp-bbcggi19-overview}} with an overview of the proof system and some
+extensions to it. {{flp-bbcggi19-valid}} defines validity circuits, the core
+component of the proof system that defines measurement validity and how
+measurements are aggregated. The proof-generation algorithm, the query
+algorithm, and the decision algorithm are defined in
+{{flp-bbcggi19-construction-prove}}, {{flp-bbcggi19-construction-query}}, and
+{{flp-bbcggi19-construction-decide}} respectively.
+
+| Parameter        | Value                    |
+|:-----------------|:-------------------------|
+| `PROVE_RAND_LEN` | `valid.prove_rand_len()` |
+| `QUERY_RAND_LEN` | `valid.query_rand_len()` |
+| `JOINT_RAND_LEN` | `valid.JOINT_RAND_LEN`   |
+| `MEAS_LEN`       | `valid.MEAS_LEN`         |
+| `OUTPUT_LEN`     | `valid.OUTPUT_LEN`       |
+| `PROOF_LEN`      | `valid.proof_len()`      |
+| `VERIFIER_LEN`   | `valid.verifier_len()`   |
+| `Measurement`    | `valid.Measurement`      |
+| `AggResult`      | `valid.AggResult`        |
+| `field`          | `valid.field`            |
+| `valid`          | Instance of `Valid` ({{flp-bbcggi19-valid}}) |
```

Comment:
I think we should move this row to the top of the table, so we get the forward reference to the definition of a validity circuit sooner. (since the whole table references parameters of validity circuits)

Reply:
Done (will do the same in the next PR in the stack)